code/src/ProjectTemplates/WinUI/CS/CSharp.WinUI.Desktop.ClassLibrary/CSharp.WinUI.Desktop.ClassLibrary.vstemplate

```diff
@@ -15,7 +15,7 @@
     <NumberOfParentCategoriesToRollUp>2</NumberOfParentCategoriesToRollUp>
     <TargetPlatformName>Windows</TargetPlatformName>
     <CreateInPlace>true</CreateInPlace>
-    <PreviewImage>WinUI.Desktop.Cs.ClassLibrary.png</PreviewImage>
```

Comment:
Why bother with this? The preview image isn't displayed anywhere. You're adding an image to the package that isn't ever shown.

Reply:
Ok, according to the documentation the element "PreviewImage" is optional: https://docs.microsoft.com/visualstudio/extensibility/previewimage-element-visual-studio-templates?view=vs-2019

Actually the current VSIX from Project Reunion includes this PreviewImage, but we will remove it.
https://github.com/microsoft/ProjectReunion/blob/main/dev/VSIX/ProjectTemplates/Desktop/CSharp/ClassLibrary/WinUI.Desktop.Cs.ClassLibrary.vstemplate